sklearn/metrics/_classification.py

```diff
@@ -1274,10 +1274,10 @@ def fbeta_score(
     The F-beta score is the weighted harmonic mean of precision and recall,
     reaching its optimal value at 1 and its worst value at 0.
 
-    The `beta` parameter determines the weight of recall in the combined
-    score. ``beta < 1`` lends more weight to precision, while ``beta > 1``
-    favors recall (``beta -> 0`` considers only precision, ``beta -> +inf``
-    only recall).
```

Comment:
I think some will still find the asymptotic behavior useful. May you still include one line about the asymptotic behavior?

Reply:
Sure. Please have a look